Override undici to ^7.24.0.

@generated
@generator: scripts/apply-override.sh --patch common/override-undici.diff --override 'direct:undici=^7.24.0' --override 'remote/package.json@global:undici=^7.24.0'
@override-package: undici@^7.24.0

Index: code-editor-src/package.json
===================================================================
--- code-editor-src.orig/package.json
+++ code-editor-src/package.json
@@ -121,7 +121,7 @@
     "open": "^10.1.2",
     "playwright-core": "1.59.0-alpha-2026-02-20",
     "tas-client": "0.3.1",
-    "undici": "^7.18.2",
+    "undici": "^7.24.0",
     "v8-inspect-profiler": "^0.1.1",
     "vscode-oniguruma": "1.7.0",
     "vscode-regexpp": "^3.1.0",
Index: code-editor-src/remote/package.json
===================================================================
--- code-editor-src.orig/remote/package.json
+++ code-editor-src/remote/package.json
@@ -42,6 +42,7 @@
     "yazl": "^2.4.3"
   },
   "overrides": {
+    "undici": "^7.24.0",
     "node-gyp-build": "4.8.1"
   }
 }
